### PR TITLE
fix migration

### PR DESCRIPTION
**Pull request by avarsava**

### CODE REVIEW

**Review by apmasell — APPROVED** [review]

**Review by djcooke — COMMENTED** [review]
Needs a release note. Could fix the previous versions' notes at the same time. Should look something like:

Unreleased: fixed Flyway migration
1.21.0: attempt to fix Flyway migration; Known Issues: Flyway migration fails on some databases
1.20.0 Known Issues: Flyway migration fails on some databases

**Comment by avarsava** [comment]
> Needs a release note. Could fix the previous versions' notes at the same time. Should look something like:
> 
> Unreleased: fixed Flyway migration
> 1.21.0: attempt to fix Flyway migration; Known Issues: Flyway migration fails on some databases
> 1.20.0 Known Issues: Flyway migration fails on some databases

Done

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+Known Issues:
+  * Flyway migrations fails on some databases
 
 # 1.20.0
 
 Changes:
-
-  * Added new project fields:                                                    
-    * samples expected                                                           
+  * Added new project fields:                                                       * samples expected
```
Comment (djcooke) [inline RELEASE_NOTES.md:17]: Line break missing

Comment (avarsava) [reply RELEASE_NOTES.md]: Fixed

**Comment by avarsava** [comment]
@djcooke checksum fix pushed

**Review by djcooke — APPROVED** [review]
Please make sure to squash when merging

**Pull request merged**